status-im/status-react
# [#14689] Link previews in chat

Fixes #14689

This PR re-introduces link previews for chat messages and requests to show link previews when a message with a support link is sent to a chat. All the required UI components are ported to a new design.

<img width="564" alt="Screenshot 2023-01-16 at 14 52 15" src="https://user-images.githubusercontent.com/911127/212694250-0dc9e850-63a7-42ac-bfe0-d5f7cd889369.png">

<img width="564" alt="Screenshot 2023-01-16 at 14 42 55" src="https://user-images.githubusercontent.com/911127/212694287-f341a769-7cb0-49d5-98d5-21d3575bfca8.png">

### Summary



Documentation change PR (review please): https://github.com/status-im/status.im/pull/xxx

### Review notes


### Testing notes


#### Platforms


- Android
- iOS

##### Functional

- chats
- link preview settings

### Steps to test


- Open Status
- Send any supported link to a chat
- Make sure the preview or a request to display preview gets displayed



status: ready
